concrete/src/File/Image/Svg/SanitizerOptions.php

```diff
@@ -0,0 +1,187 @@
odeinserted ondomnodeinsertedintodocument ondomnoderemoved ondomnoderemovedfromdocument ondomsubtreemodified ondownloading ondrag ondragdrop ondragend ondragenter ondragexit ondraggesture ondragleave ondragover ondragstart ondrop ondurationchange onemptied onenabled onend onended onendevent onerror onfocus onfocusin onfocusout onfullscreenchange onfullscreenerror ongamepadconnected ongamepaddisconnected ongotpointercapture onhashchange onheld onholding onicccardlockerror oniccinfochange onincoming oninput oninvalid onkeydown onkeypress onkeyup onlanguagechange onlevelchange onload onloadeddata onloadedmetadata onloadend onloadstart onlocalized onlostpointercapture onmark onmessage onmessageerror onmousedown onmouseenter onmouseleave onmousemove onmouseout onmouseover onmouseup onmousewheel onmozaudioavailable onmozbeforeresize onmozbrowseractivitydone onmozbrowserasyncscroll onmozbrowseraudioplaybackchange onmozbrowsercaretstatechanged onmozbrowserclose onmozbrowsercontextmenu onmozbrowserdocumentfirstpaint onmozbrowsererror onmozbrowserfindchange onmozbrowserfirstpaint onmozbrowsericonchange onmozbrowserloadend onmozbrowserloadstart onmozbrowserlocationchange onmozbrowsermanifestchange onmozbrowsermetachange onmozbrowseropensearch onmozbrowseropentab onmozbrowseropenwindow onmozbrowserresize onmozbrowserscroll onmozbrowserscrollareachanged onmozbrowserscrollviewchange onmozbrowsersecuritychange onmozbrowserselectionstatechanged onmozbrowsershowmodalprompt onmozbrowsertitlechange onmozbrowserusernameandpasswordrequired onmozbrowservisibilitychange onmozgamepadbuttondown onmozgamepadbuttonup onmozmousepixelscroll onmozorientation onmozscrolledareachanged onmoztimechange onmoztouchdown onmoztouchmove onmoztouchup onmscontentzoom onmsmanipulationstatechanged onmspointerhover onnomatch onnotificationclick onnoupdate onobsolete onoffline ononline onopen onorientationchange onoverflow onpagehide onpageshow onpaste onpause onplay onplaying onpointercancel onpointerdown onpointerenter onpointerleave onpointerlockchange onpointerlockerror onpointermove onpointerout onpointerover onpointerup onpopstate onprogress onpush onpushsubscriptionchange onratechange onreadystatechange onreceived onrepeatevent onreset onresize onresourcetimingbufferfull onresult onresume onresuming onscroll onseeked onseeking onselect onselectionchange onselectstart onsent onshow onslotchange onsmartcard-insert onsmartcard-remove onsoundend onsoundstart onspeechend onspeechstart onstalled onstart onstatechange onstatuschange onstkcommand onstksessionend onstorage onsubmit onsuccess onsuspend onsvgabort onsvgerror onsvgload onsvgresize onsvgscroll onsvgunload onsvgzoom ontext ontimeout ontimeupdate ontouchcancel ontouchend ontouchenter ontouchleave ontouchmove ontouchstart ontransitionend onunderflow onunload onupdateready onupgradeneeded onuploadprogress onuserproximity onussdreceived onversionchange onvisibilitychange onvoicechange onvoiceschanged onvolumechange onwaiting onwheel';
```

Comment:
Can we split this across a few lines?

Reply:
@KorvinSzanto how?

Reply:
Since we support PHP 5.5 I think the best option would be to implement a method that supplied these and just split the string. If you don't want to do that I think the only way is with arrays. 

This could totally just sit in a trait in a protected method if keeping the class clean is priority